liquibase-core/src/main/java/liquibase/integration/commandline/Main.java

```diff
@@ -455,10 +455,10 @@ protected void parseDefaultPropertyFiles() throws CommandLineParsingException {
      * @throws IOException if the file cannot be opened
      * @throws CommandLineParsingException if an error occurs during parsing
      */
-    private void parseDefaultPropertyFileFromResource(File potentialPropertyFile) throws IOException,
```

Comment:
Same here, seems the method's goal is to read a (local) file, not a resource-reference.